docs/usage.md

```diff
@@ -69,119 +113,379 @@ cat ./compilation.json
+### Run the analysis<a name="run-the-analysis"></a>
 ```sh
-CodeChecker analyze compilation.json -o ./reports
+CodeChecker analyze ./compile_commands.json --enable sensitive --output ./reports
 ```
-2. View the analysis results in the command line
+
+However the compilation of the project (here the example program) is performed
+by `gcc`, the CodeChecker uses `clang` to analyze sources of the project.
+During analysis, CodeChecker compiles/analyses sources again. The analysis
+process generally uses more time. If you want to speed up analysis specify a
+higher value for the `--jobs` option.
+
+In the above command the `--enable sensitive` means that a subset of checker
+are run. `sensitive` chooses a predefined "group" of checkers. The further
+info on available checkers use
+
+```sh
+# List available checkers.
+CodeChecker checkers --help
+# Show more information about the checkers.
+CodeChecker checkers --details --help
+# List profiles.
+CodeChecker checkers --profile list --details
+# List checkers which are in the sensitive profile.
+CodeChecker checkers --profile sensitive --details
+```
+commands.
+
+The `./reports` directory is the "database" of CodeChecker that allows to
+manage further working steps.
+
+### View the analysis results in the command line<a name="view-analysis-results"></a>
 ```sh
 CodeChecker parse ./reports
 ```
 
 Hint:
- You can do the 1st and the 2nd step in one round by executing `check`
+You can do the 1st and the 2nd step in one round by executing `check`
 ```sh
-cd tmux
+cd <repo root dir>/docs/examples
 make clean
-CodeChecker check -b "make" -o ./reports
+rm -rf ./reports
+CodeChecker check --build "make" --output ./reports --clean
 ```
-or to run on 22 threads
+or to run on 22 threads both the compilation and the analysis:
+
 ```sh
-CodeChecker check -j22 -b "make clean;make -j22" -o ./reports
+CodeChecker check --jobs 22 --build "make clean ; make --jobs 22" \
+    --output ./reports --clean
 ```
 
-### Cross-Compilation <a name="cross-compilation"></a>
+### Cross-Compilation<a name="cross-compilation"></a>
 Cross-compilers are auto-detected by CodeChecker, so the `--target` and the
-compiler pre-configured
-include paths of `gcc/g++` are automatically passed to `clang` when analyzing.
+compiler pre-configured include paths of `gcc/g++` are automatically passed to
+`clang` when analyzing.
 
 **Make sure that the compilers used for building the project (e.g.
 `/usr/bin/gcc`) are accessible when `CodeChecker analyze` or `check` is
 invoked.**
 
-### Incremental Analysis <a name="incremental-analysis"></a>
+## Step 3: View analysis results in command line or generate static HTML files<a name="step-3"></a>
+You can print detailed results (including the control flow) in command line by
+running:
+
+```sh
+CodeChecker parse --print-steps ./reports
+...
+Found no defects in divide.c
+[MEDIUM] src/main.c:17:9: function 'strcmp' is called without explicitly comparing result [bugprone-suspicious-string-compare]
```

Comment:
I tried to run the command above (`CodeChecker check --build "make" --output ./reports --clean`) but it didn't find this error. The reason is that it is not enabled by the default profile:
https://github.com/Ericsson/codechecker/blob/e02baa734ab58c696170c7289477b2ebee1d5b06/config/checker_profile_map.json#L123-L126
So remove this error from this output, use different profile or enable this checker with the --enable option.

Reply:
The --enable sensitive missed from the previous commands.